src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -152,7 +154,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
             {
                 IWcfService proxy = factory.CreateChannel();
                 watch.Start();
-                proxy.EchoWithTimeout("Hello");
+                proxy.EchoWithTimeout("Hello", serviceOperationTimeout);
             });
         }
         finally
```

Comment:
@StephenBonikowsky Just open a issue on our side to cross reference a bug in corefx.having a comment in code